Tools/gsc/test/sh.manifest

```diff
@@ -0,0 +1 @@
+sgx.enclave_size = 4G
```

Comment:
With Enclave size 4G, Graphene-SGX was crashing for me in `exec` test in ACC VM. It's working if I keep `sgx.enclave_size = 1G`. I've raised an issue
https://github.com/oscarlab/graphene/issues/1569#issuecomment-648058870. I've attached the standalone graphene-SGX `exec` code in the issue . Would be good if you can include `exec` with sh in GSC test cases.